src/test/scala/uk/gov/nationalarchives/downloadfiles/FileUtilsTest.scala

```diff
@@ -179,6 +181,67 @@ class FileUtilsTest extends AnyFlatSpec with MockitoSugar with ScalaFutures  {
     res.getMessage shouldEqual "GraphQL response contained errors: General error"
   }
 
+  "The addFileStatus method" should "add a file status" in {
```

Comment:
Can remove the `The` from the test title.

Reply:
Yeah, we can remove it. Actually, all other test methods start with 'The', so I did in the same way.